PROD-2346: Phone input changes

Marcisbee opened the pull request:
#### Task description ([PROD-2346](https://kaskoltd.atlassian.net/browse/PROD-2346)):

![Demo](https://media.giphy.com/media/2jarHKD0OhpHMdH6sc/giphy.gif)

- [x]  **Show country code in telephone field**
  -  Country code should be visible at all times now (currently not optional, will do this afterwards).
  -  It's no longer part of placeholder.
- [x]  **Delete leading zero if customer enters it**
  -  In output zeros are removed, but for user they are not. It is not a good practice to do this (in one case you can enter 0, but in other case you can't) as it can confuse users.
  -  Used `.charCodeAt()` instead of regexp, as it's a bit faster to do it this way.

---

#### Checklist:

- [X] I'm happy with my code
- [X] Tested on Demo app
- [x]  Tested on QA

MatissJanis commented:
btw - this project does not have proper circleci publishing to npmjs. So you can do that locally:

1. bump version number;
2. build
3. `npm publish` or possible `npm publish dist`

Marcisbee commented:
#### A bit more code!

- Tested this on QA, fixed styling a bit for general use.
- Added filter that removes country code on initial value so that when we go back one step in webapp and preset phone value with "+41..." it doesn't show "+41 +41...", as it's always present.

---

Validation with zeros works!

In output value there are no changes:

![screen shot 2018-08-01 at 10 34 40](https://user-images.githubusercontent.com/16621507/43508109-afcaa048-9577-11e8-9b00-88e00cb4a205.png) ![screen shot 2018-08-01 at 10 36 30](https://user-images.githubusercontent.com/16621507/43508123-b6f390d2-9577-11e8-903a-82fb7387fe56.png)

MatissJanis commented:
LGTM
